src/CoreWCF.Primitives/src/CoreWCF/Channels/ChannelDemuxer.cs

```diff
@@ -512,7 +508,7 @@ public async Task DispatchAsync(RequestContext context)
                 await serviceChannelDispatcher.DispatchAsync(context);
             }
 
-            public Task DispatchAsync(Message message)
+            public async Task DispatchAsync(Message message)
             {
                 throw new NotImplementedException();
             }
```

Comment:
Remove async, use Task.FromException(new NotImplementedException());

Reply:
good to know.. thanks !